src/reducers/autoCompleteReducer.js

```diff
@@ -0,0 +1,51 @@
+export default function (state = {}, action) {
```

Comment:
@Moe-Shoman Having this be a separate reducer feels right :+1: nice!  Can we have multiple forms simultaneously auto-completing?

Reply:
If `state` had the structure of `{suggestionForm: {suggestions: ...}}` rather than `{suggestionForm: ..., suggestions: ...}` then we would support multiple simultaneous completions automatically, yes, but no more than 1 per form (or form field?) (which we wouldn't want anyway)?

Reply:
How does that sound? Auto-completing something in LeapChat (once it's no longer in an iframe) and in the right panel, or some filter/search box, at the same time isn't super unlikely.

Reply:
Yeah, our current state only has one `suggestionForm`, so we shouldn't run into any problems with autocompleting two forms at the same time